client/ayon_maya/plugins/publish/extract_maya_usd.py

```diff
@@ -417,6 +423,14 @@ def filter_members(self, members):
         members = cmds.ls(cmds.sets(out_set, query=True), long=True)
         return members
 
+    def override_options(self, options: dict) -> dict:
+        options = options.copy()
+        options["exportUVs"] = False
+        options["exportColorSets"] = False
+        options["exportMaterials"] = False
+        options["exportAssignedMaterials"] = False
```

Comment:
I tested out with these two options and also do some research on the [mayaUSD](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/commands/Readme.md). It seems that these two options haven't been exposed to Python as it errors out in maya 2025.1 during extraction.
![image](https://github.com/user-attachments/assets/812fb716-72f8-4660-8204-fd71d7066fda)

![image](https://github.com/user-attachments/assets/2bf2cee4-64a6-470c-9343-8d50f88ce0cc)

Reply:
That's odd. But they are in the list here: https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/commands/Readme.md#mayausdexportcommand

Can you check which `maya-usd` addon version you're using?

Reply:
I am at 0.28.0 not the the latest 0.30.0. That could be the reason

Reply:
Thanks - yes. Seems added in 0.29.0 -> https://github.com/Autodesk/maya-usd/blob/v0.29.0/doc/CHANGELOG.md

Will take it into consideration. 👍